.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea/
+.gitignore
```

Comment:
I don't think we need to add gitignore to gitignore - it's already in there. Also, please remove protected/config and index.php from there as well.